pandas/core/missing.py

```diff
@@ -275,12 +272,7 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
             raise ImportError("Your version of Scipy does not support "
                               "PCHIP interpolation.")
     elif method == 'akima':
-        try:
```

Comment:
Akima1DInterpolater was added in SciPy 0.14, and we now require 0.19, so this additional check isn't needed anymore.